frontend/src/pages/ArtifactDetails.tsx

```diff
@@ -88,38 +140,43 @@ export default class ArtifactDetails extends Page<{}, ArtifactDetailsState> {
   }
 
   private async load(): Promise<void> {
```

Comment:
```suggestion
  private load = async (): Promise<void> => {
```
and get rid of `load.bind`